go.mod

```diff
@@ -18,10 +18,10 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/viper v1.15.0
 	github.com/strangelove-ventures/packet-forward-middleware/v6 v6.0.2
-	github.com/stretchr/testify v1.8.1
+	github.com/stretchr/testify v1.8.2
 	github.com/tendermint/tendermint v0.34.27
 	github.com/tendermint/tm-db v0.6.8-0.20221109095132-774cdfe7e6b0
-	github.com/terra-money/alliance v0.1.0-beta1
+	github.com/terra-money/alliance v0.1.1-rc.0
```

Comment:
Can we update to use `[v0.1.1](https://github.com/terra-money/alliance/releases/tag/v0.1.1)`